core/src/test/kotlin/org/stellar/anchor/api/sep/sep10/ChallengeRequestTest.kt

```diff
@@ -12,7 +12,7 @@ internal class ChallengeRequestTest {
   }
 
   @Test
-  fun of() {
+  fun `test ChallanegeRequest creation`() {
```

Comment:
What do you think of being more explicit about the class and method signature? Something like this:

```suggestion
  fun `test ChallanegeRequest.of`() {
```

Reply:
It is not proper to use `of`. It's now changed to builder pattern.